py-polars/polars/io.py

```diff
@@ -206,6 +206,24 @@ def read_csv(
     --------
     scan_csv : Lazily read from a CSV file or multiple files via glob patterns.
 
+    Examples
+    --------
+    >>> import polars as pl
+    >>> df = pl.read_csv("file.csv", sep=",") # doctest: +SKIP
+    >>> df # doctest: +SKIP
```

Comment:
We use `blackdoc` to automatically format our doc examples. In this case, I think it's failing because inline comments need to start with two spaces.

Reply:
@stinodego In the contributing.md it is stated:  
`"If you want to build the API reference locally, go to the py-polars/docs directory and run make html SPHINXOPTS=-W. The resulting HTML files will be in py-polars/docs/build/html."`

However, when i run the make command, I get the following error:

`make: *** No rule to make target `html'.  Stop.`

So then I installed sphinx using:
`pip install -U sphinx`

after which running the command again will give multiple configuration errors on modules i'm missing. Is there an overview what needs to be installed to be able to run the make command>

Reply:
You can set up your virtual environment by running `make build` in the `py-polars` directory. That will create a virtual environment with all the documentation dependencies included. If you activate this venv before you do `make html`, it should run.

I guess this is inconsistent as the rest of the Makefile runs without having to activate a venv; I'll look into improving this.